derive/src/lib.rs

```diff
@@ -3,7 +3,7 @@ use quote::quote;
 use serde_derive_internals::{attr::get_serde_meta_items, Ctxt};
 use syn::{parse_macro_input, Data, DataStruct, DeriveInput, Fields, Ident, Lit, Meta, NestedMeta};
 
-/// Parses `#[serde(skip_serializing)]`
+/// Parses `#[serde(skip_*)]`; does not work properly with `#[serde(skip_serializing_if)]`
```

Comment:
Let's explicitly allow `skip_deserializing` to avoid a possible footgun for users.